docs/sources/features/datasources/elasticsearch.md

```diff
@@ -170,11 +170,11 @@ Select the Elasticsearch data source, change to Logs using the Metrics/Logs swit
 
 ![](/img/docs/elasticsearch/explore_elasticsearch_logs_switch.png)
 
-and then enter your query using [Lucene](https://lucene.apache.org/core/2_9_4/queryparsersyntax.html) syntax into the query field
+and then optionally enter a [Lucene](https://lucene.apache.org/core/2_9_4/queryparsersyntax.html) query into the query field to filter the log messages.
```

Comment:
If you're using Elasticsearch we expect the user to be familiar with lucene. As you can see in this documentation we mention lucene in a lot of places without linking to any documentation. Your link seems to reference a really old version of lucene. If you still want to link we should do it everywhere we mention lucene and link to https://www.elastic.co/guide/en/elasticsearch/reference/7.0/query-dsl-query-string-query.html#query-string-syntax instead.